app/routes/admin/sessions/list.js

```diff
@@ -0,0 +1,108 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    switch (this.get('params.sessions_state')) {
+      case 'pending':
+        return this.i18n.t('Pending');
```

Comment:
Hi! @CosmicCoder96 Thanks for pointing out. I didn't think of it. Yes, we can definitely replace this with switch. In fact I'll replace it and update the PR. Thanks again :+1: